static/app/routes.tsx

```diff
@@ -60,12 +59,17 @@ export function makeLazyloadComponent<C extends ComponentType>(
 ) {
   // XXX: Assign the component to a variable so it has a displayname
   const RouteLazyLoad: React.FC<React.ComponentProps<C>> = props => {
-    return <SafeLazyLoad {...props} component={resolve} />;
+    // pass the organization to the RouteAnalyticsContext
+    const organization = useContext(OrganizationContext);
+    const {setOrganization, ...routeAnalytics} = useContext(RouteAnalyticsContext);
+    useEffect(() => {
+      organization && setOrganization(organization);
+    }, [organization, setOrganization]);
+    return <SafeLazyLoad {...props} {...routeAnalytics} component={resolve} />;
```

Comment:
yeah I don't... love this, but maybe it's fine. I might even move this into a hook somewhere, so you can just write this like

```suggestion
    const routeAnalytics = useRouteAnalytics();
    return <SafeLazyLoad {...props} {...routeAnalytics} component={resolve} />;
```

and then that useRouteAnalytics thing can live somewhere else

Reply:
I'm saying you don't need `RouteAnalyticsContextProvider`, it can just be a hook. there's no need for context afaict?